examples/decoding/plot_haxby_searchlight.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 import nibabel
 from nilearn import datasets
+from nilearn.image import index_img
```

Comment:
I agree it's not super consistent but in examples, import are generally close to the code where they are needed to make in order to have an interactive session feel, where you import stuff where you realise you need them.

So maybe move this import below the `### Restrict to faces and houses` comment